map/routing_manager.cpp

```diff
@@ -219,7 +221,7 @@ RoadWarningMarkType GetRoadType(RoutingOptions::Road road)
 }
 
 drape_ptr<df::Subroute> CreateDrapeSubroute(vector<RouteSegment> const & segments, m2::PointD const & startPt,
-                                            double baseDistance, double baseDepth, bool isTransit)
+                                            double baseDistance, double baseDepth, bool isTransit, bool isRuler)
```

Comment:
nit: Pass one enum instead if isTransit and isRuler?

Reply:
Replaced flags with enum